README.md

```diff
@@ -691,10 +695,25 @@ The following protocol specification features are not currently implemented:
   recorded in issue [100](https://github.com/yahoo/proxy-verifier/issues/100).
 * Only TCP is supported. There have been recent discussions about adding
   HTTP/3 support, which is over UDP, but work for that has not yet started.
+* the PROXY protocol support is limited to the `PROXY` command via TCP over IPv4 or IPv6. Therefore,
+  * No support for `AF_UNIX` socket address as either source and destination address.
+  * No support for UDP transport type.
+  * No support for `LOCAL` command(in v2 header).
 
 If there is no `protocol` node specified, then Proxy Verifier will default to
 establishing an HTTP/1 connection over TCP (no TLS).
 
+#### PROXY protocol support
+Proxy Verifier supports sending and receiving the PROXY protocol, which can be
+helpful to verify the PROXY protocol functionalities of the proxy under test.
+The feature can be enabled by specifying the `proxy-protocol` protocol node as
+outlined above. If enabled, the Verifier client would send out the PROXY
+protocol header at the beginning of the connection. Upon receiving a PROXY
+protocol header, the Verifier server would display it in the human-readable v1
+format. Note that the specification of source and destination addresses are
+supported but not required; if not specified, the Proxy Verifier client would
+send out PROXY message with the source and destination addresses matching the
+underlying socket, similar to how `curl --haproxy-protocol` behaves.
 ### Session and Transaction Delay Specification
```

Comment:
Probably good to add another space here.

Reply:
Added a linebreak.